src/indra_cogex/sources/nih_reporter/__init__.py

```diff
@@ -0,0 +1,153 @@
+import re
+from typing import Iterable
+import zipfile
+from collections import defaultdict
+import pandas
+import pystow
+from indra_cogex.sources.processor import Processor
+from indra_cogex.representation import Node, Relation
+
+
+# Regular expressions to find files of different types
+fname_regexes = {
+    "project": re.compile(r"RePORTER_PRJ_C_FY(\d+).zip"),
```

Comment:
might want to escape the `.` in the file extension since those are technically wildcards